### PR TITLE
Update RESTEasy to 6.2.0.Final

### PR DESCRIPTION
**Pull request by liweinan**

### CODE REVIEW

**Comment by liweinan** [comment]
@jamezp I plan to update the RESTEasy dependency version and do a Final release for `resteasy-extension`, and then do the upgrade in WildFly, which is required by https://github.com/wildfly/wildfly/pull/15198 wdyt?

File: cache/cache-core/pom.xml [hunk 1]
```diff
@@ -82,6 +82,12 @@
             <artifactId>junit-jupiter</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>commons-io</groupId>
+            <artifactId>commons-io</artifactId>
+            <version>2.11.0</version>
+            <scope>test</scope>
+        </dependency>
```
Comment (jamezp) [inline cache/cache-core/pom.xml:90]: What is the reason for this?

**Review by jamezp — COMMENTED** [review]
I feel like we have a chicken and egg problem. WildFly is now on 6.2.0.Beta1. Tomorrow, maybe I'll end up having to do it today, I'll do a 6.2.0.Final which will go into WildFly. We may as well move this to 6.2.0.Final as well. The problem would be the timing.

File: cache/cache-core/pom.xml (hunk 1 above)
Comment (jamezp) [reply cache/cache-core/pom.xml:90]: Ah, that makes sense now that I think about it. I removed commons-io from RESTEasy.

**Comment by liweinan** [comment]
I'll wait for the RESTEasy `6.2.0.Final` release and then go on working on this.

**Pull request merged**